app/services/participants/query.rb

```diff
@@ -60,11 +60,12 @@ def where_from_participant_id_is(from_participant_id)
     end
 
     def order_by
-      sort_order(sort:, model: User, default: { accepted_at: :asc })
+      sort_order(sort:, model: User, default: { created_at: :asc })
```

Comment:
On the sorting order.. this change rolls back the one made here? https://dfedigital.atlassian.net/browse/CPDLP-3349

Reply:
Having a rethink about this, past Mook's change which was blocked by this, I think we should be ordering on the user created at, as this is a participant/user endpoint rather than application endpoint. I think revisiting that ticket we should probably have kept it by user